pom.xml

```diff
@@ -771,6 +777,10 @@
           <id>cloudera</id>
           <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
         </repository>
+        <repository>
+          <id>hortonworks</id>
+          <url>http://repo.hortonworks.com/content/groups/public/</url>
```

Comment:
This version(2.11.0.SP5) of xercesImpl is not available in central repo, hence added this repo.

Reply:
Officially it is available here "https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/", should I add that as well?